File: tf_examples/src/broadcaster.cpp
#include<ros/ros.h>
#include<tf2_ros/transform_broadcaster.h>
#include<geometry_msgs/TransformStamped.h>

void callback(const geometry_msgs::TransformStamped &msg)
{
    static tf2_ros::TransformBroadcaster br;
    br.sendTransform(msg);
    ROS_INFO("updated tf");

}


int main(int argc,char ** argv)
{
    ros::init(argc,argv,"broadcaster");
    ros::NodeHandle n;
    ros::Subscriber sub = n.subscribe("pose",10,callback);
    ros::spin();
    return 0;
}

File: tf_examples/src/broadcaster.py
#!/usr/bin/env python

import rospy
from tf2_ros import TransformBroadcaster
from geometry_msgs.msg import TransformStamped


def callback(msg):
    br = TransformBroadcaster()
    br.sendTransform(msg)


def main():
    _ = rospy.Subscriber('pose',TransformStamped,callback)
    rospy.spin()

if __name__ == '__main__':
    rospy.init_node('broadcaster')
    try :
        main()
    except rospy.ROSException as e:
        print(e)